x/evm/keeper/querier.go

```diff
@@ -195,3 +198,30 @@ func queryAccount(ctx sdk.Context, path []string, keeper Keeper) ([]byte, error)
 	}
 	return bz, nil
 }
+
+func queryExportAccount(ctx sdk.Context, path []string, keeper Keeper) ([]byte, error) {
+	addr := ethcmn.HexToAddress(path[1])
+
+	var storage []types.GenesisStorage
+	err := keeper.CommitStateDB.ForEachStorage(addr, func(key, value ethcmn.Hash) bool {
+		storage = append(storage, types.NewGenesisStorage(key, value))
+		return false
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	res := types.GenesisAccount{
+		Address: addr,
+		Balance: keeper.GetBalance(ctx, addr),
+		Code:    keeper.GetCode(ctx, addr),
+		Storage: storage,
+	}
+
+	bz, err := json.MarshalIndent(res, "", "\t")
```

Comment:
for consistency with the other methods:
```suggestion
	bz, err := codec.MarshalJSONIndent(keeper.cdc, res)
```